test/system/github/workflows_test.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+module Github
+  class WorkflowsTest < ApplicationSystemTestCase
+    setup do
+      login_as :john
+      @team = teams(:three)
+      @app = apps(:three)
+    end
+
+    test 'visiting the index' do
+      VCR.use_cassette('system/github/workflows') do
+        visit team_app_workflows_path(@team, @app)
+        assert_selector 'a.border-indigo-500.border-b-2', text: 'Workflows'
```

Comment:
This is brittle. Could we change it to not select based on the border color?

Reply:
Yeah i'll change it, I wrote that. My thought was to assert that the correct tab was selected and highlighted but it is brittle.